External/Plugins/ResultsPanel/PluginMain.cs

```diff
@@ -19,17 +19,12 @@ public class PluginMain : IPlugin
         private String pluginHelp = "www.flashdevelop.org/community/";
         private String pluginDesc = "Adds a results panel for console info to FlashDevelop";
         private String pluginAuth = "FlashDevelop Team";
-        private DockContent pluginPanel;
+        //private DockContent pluginPanel;
         private Settings settingObject;
         private String settingFilename;
-        private PluginUI pluginUI;
+        internal PluginUI pluginUI;
```

Comment:
I see, I tried to look for references and didn't find any, but GitHub is not the best tool for that kind of operation.